### DISCUSSION TITLE
Implement widget/plugin to system bar (Ubuntu)

### DISCUSSION THREAD

**Discussion by AbdelrahmanElhosiny** [DBChoco/Muezzin#40] [General]
Is there a way to get the remaining time to show either using a widget or a plugin to the system bar or system tray ?

**Comment by DBChoco** [comment]
Assalam Alaykoum, 
what OS are we talking about please ?
In any case, at the time it is not availible on any OS, and I don't know of a way to implements this on Electron.
Sorry about that

**Reply by AbdelrahmanElhosiny** [reply]
Alaykoum Assalam,
I am using ubuntu 23.10, the system just updated and that made the gnome extension I was using to show prayer time not work anymore, I believe it's hard to implement a widget that can support more than a single os as they operate differently.

**Reply by DBChoco** [reply]
Yeah I believe I know the app you're talking about. Unfortunatly it's hard to implement something like that for me, especially considering that even if I managed to do it, only Ubuntu distros would be compatible.
Plus that would require a lot of time, which I don't currently have.

I'll keep the issue opened inshaAllah, perhaps you can research yourself how to go about it and implement it in the app, I'd be more than happy to help or verify what you write.
Or perhaps someone else could come and do it, inshaAllah we'll see.

**Reply by AbdelrahmanElhosiny** [reply]
Unfortunately, I am still a beginner in coding, I will have to study more to be able to add such functionality, inshaàllah by time  I will be able to get a hang of it.
